mount.go

```diff
@@ -404,3 +404,28 @@ func bindUnmountAllRootfs(sharedDir string, pod Pod) {
 		}
 	}
 }
+
+// getSCSIIdLun gets the SCSI id and lun, based on the index of the drive being inserted.
+// qemu code suggests that scsi-id can take values from 0 to 255 inclusive, while lun can
+// take values from 0 to 16383 inclusive. But lun values over 255 do not seem to follow
+// consistent SCSI addressing. Hence we limit to 255.
+func getSCSIIdLun(index int) (int, int, error) {
```

Comment:
Could you rename to `scsiIDLun` or similar (https://golang.org/doc/effective_go.html#Getters)?

Reply:
@jodh-intel This is not technically a getter for a struct field, but a function that derives the SCSI id and LUN based on an index.

Reply:
Ah - fair point.